_posts/en/2019-12-20-why-we-are-promoting-risks.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title:  "We are actually promoting risks in IT"
+ref: why-we-are-promoting-risks
+lang: en
+author: "Rémy Bernard, in collaboration with the IT Strategy team"
+date: "2019-12-20"
+excerpt_separator: <!--more-->
+---
+In this blog, I am claiming that IT initiatives are currently governed in a way that ends up promoting risks instead of reducing them, leads to incorrect investment decisions, dilutes our empathy towards end-users, and ultimately reduces our ability to meet our fiduciary responsibility towards public funds as required in the [Financial Administration Act](https://laws.justice.gc.ca/eng/acts/F-11/index.html).
+<!--more-->
+
+This blog will link to internal ESDC documents which are only accessible within the ESDC corporate network.
+
+## Problem Statement
+
+Governance of IT initiatives is required and I am in no way saying we need to remove it. However, the current methods seek a high level of predictability using advanced planning and rigid plan execution. This may work for physical assets like constructing a house, but not for intangibles -- like software. In addition, the transition towards Digital Government is creating a future that is highly complex and uncertain, which I argue requires a different means to govern IT investments: one that rewards an empirical cycle of trying, observing, and correcting.
+
+At ESDC, we have a [Policy on Project and Program Management (PPPM)](https://gpp-ppm.service.gc.ca/sites/pwa/ESDCKnowledgeRepository/All%20Documents/2-01PL-182E-Project%20and%20Programme%20Management%20Policy%20on-withBanner-v1.0.pdf) that complies with the Treasury Board (TB) [Policy on Planning and Management of Investments](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32593) with one of its objectives being ensuring *accountability and success of the Portfolio of projects and programmes with a ***specific focus on the benefits of investments*** in projects and programmes*. This ESDC Policy acts as the authoritative means under which all projects and programmes within ESDC are managed. The [Project Lifecycle framework (PMLC)](https://gpp-ppm.service.gc.ca/sites/pwa/ESDCKnowledgeRepository/All%20Documents/02-05G-271E-ProjectLifeCycle-v4.0.pdf) is used to guide management of projects according to their classification by size (which is based on cost, risk, capability, and complexity, though the cost variable is really driving this).
```

Comment:
"though the cost variable is really driving this"

Not entirely sure what this is intending to say ... ?

Reply:
reworded to "though its fair to say the cost variable carries more weight than the others"

when project needs to be categorized, we are required to look at 4 variables (TBS has a Project Complexity and Risk Assessment Tool - PCRA) but what usually stands out more and is much more tangible is costs.

PCRA link as FYI: https://www.canada.ca/en/treasury-board-secretariat/services/information-technology-project-management/project-management/project-complexity-risk-assessment-tool.html